packages/aws-cdk-lib/aws-dynamodb/README.md

```diff
@@ -1,6 +1,704 @@
 # Amazon DynamoDB Construct Library
 
 
+Here is a minimal deployable DynamoDB `GlobalTable` definition:
```

Comment:
Don't we need a stability header at the top?

I would also like a block quote   style admonition at the top explaining that there are 2 Table constructs, and that `GlobalTable` is the new preferred one, even for single-region tables.

And maybe a link to the original README that explains `Table` (can be copied to a separate file)

Reply:
Updated.